src/main/java/com/rocana/lucene/codec/v1/RocanaLucene50PostingsFormat.java

```diff
@@ -40,6 +41,25 @@
 import org.apache.lucene.util.packed.PackedInts;
 
 /**
+ * Fork of Lucene's {@link com.rocana.lucene.codec.v1.RocanaLucene50PostingsFormat}.
+ *
+ * Why we forked:
+ *   - We need to 'hook in' {@link RocanaBlockTreeTermsReader}, which has the performance
+ *     improvement.
+ *
+ * What changed in the fork?
+ *   - {@link #fieldsProducer(SegmentReadState)} returns a fork: {@link RocanaBlockTreeTermsReader}
+ *     rather than the normal Lucene class.
+ *   - Removed trailing whitespace.
+ *   - Changed these javadocs.
+ *
+ * To see a full diff of changes: compare this version to this file's initial commit (which
+ * is the original file from Lucene).
+ *
+ * See ROCANA-8229.
```

Comment:
Same comment here about referring to internal only JIRA tickets in open sourced code.